packages/botonic-react/index.d.ts

```diff
@@ -0,0 +1,115 @@
+import * as React from 'react';
+import * as core from '@botonic/core';
+
+export interface MessageProps{
+  type? : string;
+  from? : string;
+  delay? : number;
+  typing? : number;
+  children : any;
+  json: any;
+  style: any;
+}
+export class Message extends React.Component<MessageProps, any> {}
+
+
+//TODO inherit from MessageProps?
+export interface TextProps{
+  delay?: number;
+  typing?: number;
+}
+export class Text extends React.Component<TextProps, any> {}
+
+export interface ButtonProps{
+  payload?: string;
+  url?: string;
+}
+export class Button extends React.Component<ButtonProps, any> {}
+
+export interface ButtonProps{
```

Comment:
This line is the same as line23? Maybe its ReplyProps? If it's ReplyProps it should have an optional variables called path as ButtonProps

Reply:
good spot! fixed